tajo-common/src/test/java/org/apache/tajo/datum/TestArithmeticOperator.java

```diff
@@ -40,7 +40,7 @@ public TestArithmeticOperator(String option) {
     this.option = option;
   }
 
-  @Parameters
```

Comment:
Yes, I can't see a better way now. Please refer to https://github.com/junit-team/junit4/wiki/Parameterized-tests.